### ISSUE TITLE
Connect to the Dockerized Cassandra cluster on a single machine

### ISSUE THREAD

**Issue by nephilimboy** [gocql/gocql#1312]
Hi 
I have a Cassandra cluster using the **official dockerized Cassandra** (https://hub.docker.com/_/cassandra) **on a single machine(host)**. so I mapped each dockerized Cassandra's "9042" port to the different host's port. for example, Cassandra container #1 "9042" port is mapped to the host's port "32869" and Cassandra container #2 "9042" port is mapped to the host's port "32870" and so on. the question is: how can I assign different ports with the same IP address when Im using `gocql.NewCluster`
**so basically what i need is something like this** :
`cluster := gocql.NewCluster("192.168.1.1:32869", "192.168.1.1:32870", "192.168.1.1:32871", ...)`
is it possible to do this using gocql library (and how)?
Thanks in advance

**Comment by alourie** [comment]
@nephilimboy that's exactly how it works. When you create a cluster this way, then you create a new session. When the new session is created, the during the initialisation phase the list of hosts is parsed and each address/port pair will end up creating an appropriate HostInfo object and all will work automagically.

If you want to follow the code, start here: https://github.com/gocql/gocql/blob/master/session.go#L176 and follow along.

**Comment by martin-sucha** [comment]
Closing because of inactivity - the question seems to be answered.
